consensus/ising/proposerservice.go

```diff
@@ -136,19 +145,22 @@ func (ps *ProposerService) HandleBlockForking() {
 	for {
 		select {
 		case <-timer.C:
-			//send ping to all neighbors periodically
-			nodes := ps.GetPersistedNode(nil)
-			currentHeight := ledger.DefaultLedger.Store.GetHeight()
-			currentHash := ledger.DefaultLedger.Store.GetCurrentBlockHash()
-			ps.forkCache = NewForkCache(currentHeight, currentHash)
-			// ping neighbors with local current height
-			pingMsg := NewPing(MaxUint32) // ping remote with MaxUint32 for get latest hash
-			ps.SendConsensusMsg(pingMsg, nodes)
-
-			// wait response from neighbor and handle rollback if needed
-			time.Sleep(WaitingForProbeFinished)
-			if ps.forkCache.AnalyzePingResp() {
-				ps.ProcessRollback()
+			// Don't check forked during block syncing
+			if protocol.SyncStarted != ps.localNode.GetSyncState() {
```

Comment:
is " == PersistFinished"  more suitable?

Reply:
The SyncFinished is **abnormal state**. It means that Sync routine Finished but can't switch to consensus state. Usually, it due to the cached blocks can't append to ledger. (prev header not found)
SyncFinished state node should be checked its ledger forked or not.

Reply:
@NoelBright I have another solution.
"Exit if PersistCachedBlock fail" instead of "Check ledger forked at SyncFinished state"
PR code was updated.  What do you think of？